submissions/OwoeyeGbenga/landing_page/signup.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Glitz Stores</title>
+</head>
+<body>
+    <header class="w-full">
```

Comment:
Great  Job Gbenga!!

I noticed that the header tag is been repeated in your files. You can create a class in the stylesheet file to do that instead of having to copy and paste each time you want to use it.

Reply:
Thanks, Kunmi. 
How do I go about that??. I can't seem to find a way around it